src/tests/integration_tests/test_plan_critics.py

```diff
@@ -22,20 +24,24 @@
 user_description1 = """Sahar is skilled in astronomy and astrophysics. She has a PhD in those areas specializing in star formation.
 Her expertise lies in developing and implementing models to study the formation mechanisms for galaxies and stars and applying computer vision algorithms to galaxy spectrum data.
 She can answer questions about general astrophysics topics and also in depth technical details of modelling and galaxy and star formation questions and numerical methods related to these areas.
-"""
+"""  # noqa E501
 
 user_description2 = """Amir is skilled in quantum physics, quantum information, quantum computing, quantum optics, quantum simulations, and atomic physics. He has a PhD in those areas specializing in experimental implementation of quantum computing gates.
 His expertise lies in developing models to study the behavior of quantum optical systems, and building experimental setups to test out various hypothesis using data driven techniques.
 He can answer questions about general quantum mechanics topics and also in depth technical details of modelling and implementing quantum optics experiments.
-"""
+"""  # noqa E501
 
 
-def test_planning():
-    objective = "Write a proposal for estimating the maximum wind speed of a tropical cyclone using satellite imagery"
-    config = OrchestratorConfig(llm_name='gpt-4')
+@pytest.mark.external_api
+def test_planning_successful(get_llm):
```

Comment:
Add `# noqa: F811` to this line and similar test function definitions to suppress the flake8 lint warning about unused imports.